### ISSUE TITLE
Change the format that refactorings.history uses so that it has more structure

### ISSUE THREAD

**Issue by vazexqi** [vazexqi/CodingSpectator#12]
Right now the file stores all the information into a the comment attribute in the XML node. This is very messy and makes it hard to parse.

We should create our own nodes.

**Comment by reprogrammer** [comment]
We put our additional information as new (key, value) of the arguments map rather than appending everything into the comment attribute. This issued is resolved in 746fb83697dd30e95ba773cb4e1b2bcd60243417
